index.js

```diff
@@ -92,25 +91,30 @@ function prepareOptions (options) {
 
 function registerPipeline (options, pipeline) {
 
+  debug('registering pipeline', pipeline)
+
   var bus = options.bus;
 
   var firstHandler = pipeline.handlers[0];
 
   var isAck = firstHandler.ack;
   var isListen = firstHandler.listen !== undefined;
-  var hasQueueNameSpecified = firstHandler.queueName !== undefined;
+  // var hasQueueNameSpecified = firstHandler.queueName !== undefined;
 
   var method = (isListen) ? 'listen' : 'subscribe';
-  var queueName = hasQueueNameSpecified ? firstHandler.queueName :
-        options.queuePrefix !== undefined ? util.format('%s-', queueName) : firstHandler.routingKey;
+  // var queueName = hasQueueNameSpecified ? firstHandler.queueName :
+  //       options.queuePrefix !== undefined ? util.format('%s-', queueName) : firstHandler.routingKey;
 
-  var queueName = firstHandler.queueName;
+  // var queueName = firstHandler.queueName;
```

Comment:
and what's this about?

Reply:
https://github.com/mateodelnorte/servicebus-register-handlers/pull/11

Reply:
Looks like an old mis-merge that made it through, but never caused errors because the variables were being overridden. Looks like merging this will close that PR.